client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -138,41 +171,54 @@ class RemoveDomainDialog extends Component {
 						/>
 					) }
 				</FormFieldset>
-				<FormFieldset>
-					<FormLabel>
-						<FormCheckbox name="agree" value="true" onChange={ this.onAgreeChange } />
-						<span>
-							{ translate( 'I understand that this is permanent and irreversible.', {
-								context: 'Domain name',
-							} ) }
-						</span>
-					</FormLabel>
-
-					{ this.state.showErrors && ! this.state.userAgreed && (
-						<FormInputValidation
-							text={ translate(
-								"You need to confirm that you understand the action you're performing."
-							) }
-							isError
-						/>
+				<p>
+					{ translate(
+						'We will delete the domain name. Any services related to the domain will cease to function. Be sure you wish to proceed.'
```

Comment:
I'm not sure, but maybe using the passive voice in the first sentence here would be better? Something like "This domain name will be deleted and any services related to it will cease to function. Are you sure you wish to proceed?".

Reply:
another tiny further tweak: 

"This domain name will be deleted. Any services related to it will cease to function. Are you sure you wish to proceed?"